WcaOnRails/spec/lib/results_validators/scrambles_validator_spec.rb

```diff
@@ -85,6 +86,31 @@
         end
       end
 
+      it "correctly (in)validates multiple groups for 333fm" do
+        FactoryBot.create(:round, competition: competition3, event_id: "333fm", format_id: "m")
+
+        [Result, InboxResult].each do |model|
+          result_kind = model.model_name.singular.to_sym
+          FactoryBot.create(result_kind, :mbf, competition: competition3)
+        end
+
+        # Create two groups in fmc:
+        create_scramble_set(3, competitionId: competition3.id, eventId: "333fm", groupId: "A")
+        create_scramble_set(3, competitionId: competition3.id, eventId: "333fm", groupId: "B")
+
+        expected_warnings = [
+          RV::ValidationWarning.new(:scrambles, competition3.id,
+                                    SV::MULTIPLE_FMC_GROUPS_WARNING,
+                                    round_id: "333fm-f"),
+        ]
+
+        validator_args.each do |arg|
```

Comment:
The test doesn't actually work and that's because the validator args don't mark `competition3` for validation:
https://github.com/thewca/worldcubeassociation.org/blob/a4eec2839b8a353cca8b2feab58e684ebccf30db/WcaOnRails/spec/lib/results_validators/scrambles_validator_spec.rb#L20

Reply:
yea, I had some other issues and decided to wait a day to fix them. Finally got the tests good though!